src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js

```diff
@@ -20,39 +20,69 @@ describe('UploadVideo > UploadSide', () => {
 
   const file = new File([''], 'filename');
 
-  beforeEach(() => {
-    jest.useFakeTimers();
-    jest.clearAllMocks();
-  });
-
   const createWrapper = options => {
     return shallowMount(UploadSide, {
       localVue,
       i18n,
       sync: false,
       propsData: {
-        documentType: DOC_TYPES.SELFIE,
+        documentType: DOC_TYPES.PASSPORT,
```

Comment:
upload side only for selfie, no need to set passport

Reply:
done